### PR TITLE
Update realm. The old realm didn't work because of incorrect client s…

### PR DESCRIPTION
**Pull request by kirankt**
Update realm. The old realm didn't work because of incorrect client scopes

### CODE REVIEW

**Comment by RothAndrew** [comment]
When will this merge? We need it for a deployment to avoid a manual step in the process

**Review by UnicornChance — APPROVED** [review]

File: idam/realm/default-realm.json [hunk 1]
```diff
@@ -489,14 +519,15 @@
...
-      "name": "<h1>Developer</h1><h4>Account Login</h4>",
+      "name": "<h1>UDS IDAM</h1><h4>Account Login</h4>",
       "description": "",
       "rootUrl": "${authBaseUrl}",
       "baseUrl": "/realms/###ZARF_VAR_REALM###/account/",
       "surrogateAuthRequired": false,
       "enabled": true,
       "alwaysDisplayInConsole": false,
       "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
```
Comment (rjferguson21) [inline idam/realm/default-realm.json:530]:
```suggestion
```

File: idam/realm/default-realm.json [hunk 2]
```diff
@@ -521,13 +554,14 @@
...
       "clientId": "account-console",
-      "name": "${client_account-console}",
+      "name": "<h1>UDS IDAM</h1><h4>Account Login</h4>",
       "rootUrl": "${authBaseUrl}",
       "baseUrl": "/realms/###ZARF_VAR_REALM###/account/",
       "surrogateAuthRequired": false,
       "enabled": true,
       "alwaysDisplayInConsole": false,
       "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
```
Comment (rjferguson21) [inline idam/realm/default-realm.json:564]:
```suggestion
```

File: idam/realm/default-realm.json [hunk 3]
```diff
@@ -569,6 +604,7 @@
...
```
Comment (rjferguson21) [inline idam/realm/default-realm.json:607]:
```suggestion
```

File: idam/realm/default-realm.json [hunk 4]
```diff
@@ -596,6 +634,7 @@
...
```
Comment (rjferguson21) [inline idam/realm/default-realm.json:637]:
```suggestion
```

File: idam/realm/default-realm.json [hunk 5]
```diff
@@ -652,6 +696,7 @@
...
```
Comment (rjferguson21) [inline idam/realm/default-realm.json:699]:
```suggestion
```

**Pull request merged**